docs/perf-testing/README.md

```diff
@@ -0,0 +1,181 @@
+This document outlines the instructions for performance testing using [Kwok](https://kwok.sigs.k8s.io/) for the Kyverno 1.10 release.
+
+# Pre-requisite
+
+## Install etcdctl
+
+```sh
+ETCD_VER=v3.4.13
+
+# choose either URL
+GOOGLE_URL=https://storage.googleapis.com/etcd
+GITHUB_URL=https://github.com/etcd-io/etcd/releases/download
+DOWNLOAD_URL=${GOOGLE_URL}
+
+rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+rm -rf /tmp/etcd-download-test && mkdir -p /tmp/etcd-download-test
+
+curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /usr/local/bin --strip-components=1
+rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+
+etcd --version
+etcdctl version
+```
+
+More details for etcdctl installation can be found [here](https://github.com/etcd-io/etcd/releases/tag/v3.4.13).
+
+## Download k3d:
+```sh
+wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
+```
+
+More details for k3d installation can be found [here](https://k3d.io/v5.4.9/#install-script).
+
+# Create a base cluster using K3d
+
+To quickly try out the scaling test, you can use the following command to create the k3d cluster with 3 workers:
+```sh
+k3d cluster create --agents=3  --k3s-arg "--disable=metrics-server@server:*" --k3s-node-label "ingress-ready=true@agent:*"
+```
+
+To set up embedded etcd for the K3s cluster, follow instructions below. This is used when getting objects from etcd. 
+
+```sh
+k3d cluster create scaling --servers 3 --agents=15 --k3s-arg "--disable=metrics-server@server:*" --k3s-node-label "ingress-ready=true@agent:*" 
+```
```

Comment:
How does this command enabled embedded etcd rather than SQLite?

Reply:
>The first server to be created will use the --cluster-init flag...

https://k3d.io/v4.4.8/usage/multiserver/


`--servers 3` creates a 3-server (multi-server) cluster, and the flag `--cluster-init` configures the cluster to use the embedded etcd.

Reply:
When you create a multi-server cluster with k3d, it creates the first server with `--cluster-init` (this flag is for k3s) which internally enables the embedded etcd, and the rest of the servers are joined when the first one is up.

Reply:
Ok